submissions/Barasii/a-tiny-JS-world-OOP/index.js

```diff
@@ -0,0 +1,86 @@
+class Inhabitant {
+   constructor(species, name, gender, legs, hands, saying, friends) {
+      this.species = species;
+      this.name = name;
+      this.gender = gender;
+      this.legs = legs;
+      this.hands = hands;
+      this.saying = saying;
+      this.friends = [];
+   }
+
+   toString() {
+      return Object.keys(this).map(prop => `${this[prop]}`).join('; ');
+      }
+
+   callFriends(friends) {         
+      this.friends = friends;
+      }
+}
+
+const Meow = superclass =>
+   class extends superclass {
+      constructor(species, name, gender, legs, hands, saying, friends) {
+         super(species, name, gender, legs, hands, saying, friends);
+         this.saying = 'mur-mur';
```

Comment:
Why not passing `mur-mur` straight at constructor call so that base class sets it for you?

Reply:
Done

Reply:
@Barasii cannot see any new commits. Did I miss anything?

Reply:
@OleksiyRudenko,
I corrected the situation